sensorimotor/sensorimotor/temporal_pooler.py

```diff
@@ -379,66 +383,69 @@ def reset(self):
                                             dtype=realDType)
     self._boostFactors = numpy.ones(self._numColumns, dtype=realDType)
 
-  def compute(self, inputVector, learn, activeArray, burstingColumns,
-              predictedCells):
+
+  def compute(self, inputVector, isLearning, activeArray, burstingColumns,
+              tmCorrectlyPredicted):
     """
     This is the primary public method of the class. This function takes an input
     vector and outputs the indices of the active columns.
 
-    New parameters defined here:
-    ----------------------------
-    @param burstingColumns:
-      A numpy array with numColumns elements. A 1  indicates that the column is
-      currently bursting.
-
-    @param predictedCells:
+    @param inputVector Here, the input is the active cells from the TM
+    @param isLearning Boolean specifying whether learning is on
+    @param activeArray An array representing the active columns produced by this
+                      method
+    @param burstingColumns
+      A numpy array with numColumns elements where each 1 represent a
+      currently bursting column in the TM.
+    @param tmCorrectlyPredicted
```

Comment:
is the prefix `tm` useful? wasn;t it possible to call the code w/o using temporal memory?